repository-data/webfiles/src/main/resources/site/freemarker/common/macro/sections/tableauLookupSection.ftl

```diff
@@ -180,34 +179,30 @@
                         }
                 }));
             }
-
-            function postcodeApiUrl(input) {
-                function _formatPostcode() {
-                    if ((typeof input === 'string' || input instanceof String) && input.length >= 5) {
-                        var postcode = input.split(" ").join("");;
-                        postcode = postcode.toUpperCase();
-                        return postcode.substr(0, postcode.length - 3) + "/" + postcode.substr(postcode.length - 3);
-                    } else {
-                        return null;
-                    }
+            function formatPostcode(postcode, divider) {
+                if ((typeof postcode === 'string' || postcode instanceof String) && postcode.length >= 5) {
+                    var p = postcode.split(" ").join("");
```

Comment:
I can see it was there previously, but a better way to remove spaces would probably be, `postcode.replace(/ /g, "")`

Reply:
Many thanks for the suggestion.